docs/create-packages/Symbol-Packages-V2.md

```diff
@@ -0,0 +1,96 @@
+
+Nuget.exe pack and msbuild /t:pack has an additional command line property called ```-SymbolsPackageFormat```. Since the symbols package will not be produced by default (as per current behavior), this option will only be applicable if passed along with ```-Symbols``` in nuget.exe or ```--include-symbols``` in dotnet.exe. SymbolsPackageFormat property can have one of the two values: snupkg, symbols.nupkg. The default value of this property would be snupkg.
+The option symbols.nupkg is only being provided for compat reasons with the current behavior. Using this in conjunction with -Symbols or --include-symbols will ensure the symbols.nupkg is created in the same way as it is today.
+
+
+## Creating a symbol package
+
+A snupkg symbol package can be created from a .nuspec file or from a .csproj file. Nuget.exe and dotnet.exe are both supported. When the options ```-Symbols -SymbolPackageFormat snupkg``` are used on the nuget.exe pack command a .snupkg file will be created in additon to the .nupkg file.
+
+Example commands to create .snupkg files
+```
+dotnet.exe pack MyPackage.csproj --include-symbols -p:SymbolPackageFormat=snupkg
+
+nuget.exe pack MyPackage.nuspec -Symbols -SymbolPackageFormat snupkg
+
+nuget.exe pack MyPackage.csproj -Symbols -SymbolPackageFormat snupkg
+```
+
+## Symbol package structure
+
+The .nupkg file would be exactly the same as it is today, but the .snupkg file would have the following characteristics:
+
+1) The .snupkg will have the same id and version as the corresponding .nupkg.
+2) The .snupkg will have the exact folder structure as the nupkg for any DLL or EXE files with the distinction that instead of DLLs/EXEs, their corresponding PDBs will be included in the same folder hierarchy. Files and folders with extensions other than PDB will be left out of the snupkg.
+3) The .nuspec file in the .snupkg will also specify a new PackageType as follows:
+``` 
+<packageTypes>
+  <packageType name="SybmolsPackage"/>
+</packageTypes>
+```
+4) If an author decides to use a custom nuspec to build their nupkg and snupkg, the snupkg should have the same folder hierarchy and files detailed in 2).
+5) ```authors``` and ```owners``` field will be excluded from the snupkg's nuspec.
+
+
+## Nuget.org symbol package constraints
+The symbol packages supported on nuget.org have the following contraints
+
+- Only the following file extensions are allowed to be added to a symbol package. ```.pdb,.nuspec,.xml,.psmdcp,.rels,.p7s```
+- Only managed [Portable pdbs](https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md) are currently supported on nuget symbol server.
+- The pdbs and associated nupkg dlls need to be build with compiler version 15.8 or above (see [pdb crypto hash](https://github.com/dotnet/roslyn/issues/24429))
```

Comment:
build -> built 
Should we say "with the compiler in Visual Studio version 15.8 or above" ?